iOS Example Frame/Package.swift

```diff
@@ -0,0 +1,5 @@
+// swift-tools-version:5.6
+// THIS IS TO EXCLUDE SAMPLE PROJECTS FROM PACKAGE DISTRIBUTION
+import PackageDescription
+let package = Package()
```

Comment:
How does just creating a Package instance EXCLUDE SAMPLE PROJECTS FROM PACKAGE DISTRIBUTION ?

Reply:
Seems like this is the only solution. I don't know the `how` part 🙂  @deepesh-vasthimal-cko

Reply:
Whatever the folder this empty package manifest is added, it gets excluded from the distribution. Was this your question?

Reply:
My question was how does declare just an instance of the package and not using it help us in EXCLUDE SAMPLE PROJECTS FROM PACKAGE DISTRIBUTION

`let package = Package()`

Reply:
I think SPM excludes it when we do it this way but can't know the inner logic. Tested and it works.